kolibri/content/api.py

```diff
@@ -0,0 +1,219 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
+from uuid import UUID
+import os
+from bulk_update.helper import bulk_update
+from django.core.files import File as DjFile
+from functools import wraps
+
+"""helper funcitons"""
+
+def is_valid_uuid(uuid_to_test, version=4):
+    """
+    Check if uuid_to_test is a valid UUID.
+
+    Parameters
+    ----------
+    uuid_to_test : str
+    version : {1, 2, 3, 4}
+
+    Returns
+    -------
+    `True` if uuid_to_test is a valid UUID, otherwise `False`.
+
+    Examples
+    --------
+    >>> is_valid_uuid('c9bf9e57-1685-4c89-bafb-ff5af830be8a')
+    True
+    >>> is_valid_uuid('c9bf9e58')
+    False
+    """
+    try:
+        uuid_obj = UUID(uuid_to_test, version=version)
+    except ValueError:
+        return False
+
+    return str(uuid_obj) == uuid_to_test
+
+
+"""ContentDB API methods"""
+
+def can_get_content_with_id(func):
+    """
+    decorator function for returning ContentMetadata object when giving content id
+    it can take keyword argument/s "content" or "content1" and "content2"
+    """
+    @wraps(func)
+    def wrapper(channel_id=None, **kwargs):
+        content = kwargs.get('content')
+        content1 = kwargs.get('content1')
+        content2 = kwargs.get('content2')
```

Comment:
I think my other comment got killed by the changes, anyways: Try explaining why it receives exactly content, content1, and content2? Which is which? Could it perhaps instead receive a single arg containing a list?